sdk-local-component-map.js

```diff
@@ -67,7 +63,26 @@ const localSdkComponentMap = {
   HMRC_ODX_GDSTaskListTemplate: HmrcOdxGdsTaskListTemplate,
   HMRC_ODX_sectionBased: HmrcOdxSectionBased,
   HMRC_ODX_GDSCheckAnswersPage: HmrcOdxGdsCheckAnswersPage,
-  SimpleTableManual: SimpleTableManual
+  SimpleTableManual: SimpleTableManual,
+
+  // INFRA:
+  // AssignmentCard: InfraAssignmentCard,
+  // ActionButtons: InfraActionButtons,
+  // Assignment: InfraAssignment,
+  // FlowContainer: InfraFlowContainer,
+  // ViewContainer: InfraViewContainer,
+  // View: InfraView,
```

Comment:
By commenting out, we go back to the bare bones of PEGA elements. i.e. using material UI